packages/shopify-cart/README.md

```diff
@@ -222,16 +222,17 @@ const { cart, userErrors, errors } = await cartClient.cartLinesRemove({
 
 _Updates lines on an existing Shopify cart._
 
-**Accepts**: params - an object containing the `cartId` (`string`) of interest, and the `lines` (`array`) to update. For more information on the cart `lines`, checkout Shopify's [`CartLineUpdateInput`][shopify-cart-line-update-input]
+**Accepts**: params - an object containing the `cartId` (`string`) of interest, and the `lines` (`array`) to update. For more information on the cart `lines`, checkout Shopify's [`CartLineUpdateInput`][shopify-cart-line-update-input]. Note that our lines parameter accepts a different value than Shopify's by accepting `nacelleEntryId`.
```

Comment:
I think this could be reworded to improve clarity:

```suggestion
**Accepts**: params - an object containing the `cartId` (`string`) of interest, and the `lines` (`array`) to update. For more information on the cart `lines`, checkout Shopify's [`CartLineUpdateInput`][shopify-cart-line-update-input].  Note that the `lines` parameter differs from Shopify [`CartLineInput`][shopify-cart-line-input] in that it expects the variant's `nacelleEntryId` instead of a Shopify `merchandiseId`.
```